javascript/stacksAndQueues/queueWithStacks/README.md

```diff
@@ -0,0 +1,13 @@
+# Challenge Summary
+Implement a Queue using two Stacks
+
+## Challenge Description
+Create a `PseudoQueue` class without using an existing Queue. Create your class with the following methods:
+- `enqueue(value)` which inserts value into the PseudoQueue, using a first-in, first-out approach.
+- `dequeue()` which extracts a value from the PseudoQueue, using a first-in, first-out approach.
+
+## Approach & Efficiency
+My partners and I whiteboarded for an hour, and didn't get as far ad we'd hoped to. We were contemplatng how to integrate some of the other methods into our solution and ran out of time. 
+
+## Solution
+![Queue with Stacks](../assets/queue-with-stacks.png)
```

Comment:
You look like you are off to a really great start! I would love to see code and tests added!